lms/validation/_api.py

```diff
@@ -84,10 +84,11 @@ class APICreateAssignmentSchema(PyramidRequestSchema):
 
     class Content(Schema):
         class File(Schema):
-            display_name = fields.Str(required=True)
-            id = fields.Int(required=True)
-            updated_at = fields.Str(required=True)
-            size = fields.Int(required=True)
+            class Meta:
+                unknown = EXCLUDE
+
+            # VALIDATE STARTS WITH our schemas
+            id = fields.Str(required=True)
```

Comment:
There doesn't seem to be a builtin `startswith()` validator (see the [builtin validators](https://marshmallow.readthedocs.io/en/stable/marshmallow.validate.html)) so I think you'd just add a `@validates` method: https://marshmallow.readthedocs.io/en/stable/quickstart.html#field-validators-as-methods. You can also write your own reusable validator functions but I doubt that'd be worthwhile for something this simple